CMakeLists.txt

```diff
@@ -159,7 +159,7 @@ add_test(unit
   test_units -q)
 add_test(integration_hgwosca
   test_integration_hgwosca -q)
-# add_test(integration_pengu
-  # test_integration_pengu -q)
+add_test(integration_pengu
+  test_integration_pengu -q)

```

Comment:
just out of curiousity, what does -q do?

Reply:
suppress output